tests/unit/AvirWrapperTest.php

```diff
@@ -15,6 +15,7 @@
 use Test\Util\User\Dummy;
 
 class AvirWrapperTest extends TestBase {
+
```

Comment:
@patrickjahns violates our phpcs style rules ;)

Reply:
Will add code-style in a seperate PR :-)

Reply:
@patrickjahns there is no need to add an empty line here anyway. As there are no other changes in this file.